packages/odata-service-inquirer/src/prompts/datasources/sap-system/new-system/service-helper.ts

```diff
@@ -58,26 +60,53 @@ const createServiceChoices = (serviceInfos?: ODataServiceInfo[]): ListChoiceOpti
     return choices.sort((a, b) => (a.name ? a.name.localeCompare(b.name ?? '') : 0));
 };
 
+/**
+ * Logs the catalog reuest errors.
+ *
+ * @param requestErrors catalog request errors
+ */
+function logErrorsForHelp(requestErrors: Record<ODataVersion, unknown> | {}): void {
+    // Log the first error only
+    const catalogErrors = Object.values(requestErrors);
+    if (catalogErrors.length > 0) {
+        forEach(catalogErrors, (error) => {
```

Comment:
Just a query, why the use of `lodash` forEach here?

Reply:
Auto-import 😮‍💨